webui/react/src/pages/Clusters/utils.ts

```diff
@@ -0,0 +1,4 @@
+export interface JobQueuedTimeChartSeries {
```

Comment:
I don't think that you're using this file or interface anymore (only other mention is commented out)

Reply:
Thank you for pointing that out, it's been fixed.